src/components/Audio/Audio.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faVolumeUp, faSpinner } from "@fortawesome/free-solid-svg-icons";
+
+import "./Audio.scss";
+
+interface LocalProps {
+    url: string;
+}
+type Props = LocalProps;
+
+interface LocalState {
+    play: boolean;
+}
+
+class AudioComponent extends React.Component<Props, LocalState> {
+    private audioElem: HTMLAudioElement;
+    constructor(props) {
+        super(props);
+        this.state = {
+            play: false
+        };
+        this.audioElem = new Audio(this.props.url);
+        this.audioElem.onended = this.stop;
+    }
+
+    play = () => {
+        this.setState({ play: true });
+        this.audioElem.play();
+    }
+    
+    pause = () => {
+        this.setState({ play: false });
+        this.audioElem.pause();
+    }
+    
+    stop = () => {
```

Comment:
would prefer if these function were suffixed with `Playback` so `startPlayback`, `pausePlayback`, `stopPlayback`. Easier to understand imo

Reply:
Sure, makes sense